torchci/scripts/check_alerts.py

```diff
@@ -181,8 +183,14 @@ def fetch_alerts(alert_repo: str, labels: List[str]) -> List[Any]:
             headers=headers,
         )
         r.raise_for_status()
+
         data = json.loads(r.text)
-        return data["data"]["repository"]["issues"]["nodes"]
+        # Return only alert belonging to the target repo and branch
+        return list(filter(
+            lambda alert: f"Recurrently Failing Jobs on {repo} {branch}"
```

Comment:
nit: list comprehensions are preferred ([Channeling my inner Nikita](https://github.com/pytorch/test-infra/pull/2109#discussion_r1096538566) 😀)

Reply:
TIL, the nuance :)